test/integration/user_can_see_if_item_is_retired_test.rb

```diff
@@ -9,21 +9,13 @@ class UserCanSeeIfItemIsRetiredTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test 'user can see retired item status on chip show pages' do
+  test 'user can NOT see retired item status on chip show pages' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.